kits/automation/time-series-preprocessor/.env.example

```
@@ -0,0 +1,4 @@
+TIME_SERIES_PREPROCESSOR="Your Flow ID from Lamatic Studio"
+LAMATIC_API_URL="Your API Endpoint URL"
+LAMATIC_PROJECT_ID="Your Project ID"
+LAMATIC_API_KEY="Your API Key"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)